introduction.md

```diff
@@ -1,2 +1,9 @@
 Hi, may I enter the dojo please?
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?